x-pack/plugins/rollup/server/routes/api/jobs.js

```diff
@@ -144,7 +135,7 @@ export function registerJobsRoute(server) {
           .then(() => ({ success: true }));
       } catch(err) {
         // There is an issue opened on ES to handle the following error correctly
-        // https://github.com/elastic/elasticsearch/issues/39845
+        // https://github.com/elastic/elasticsearch/issues/42908
```

Comment:
I had to create a new issue here because it looks like the old one didn't capture this case.